contracts/mock/MockERC20WithFee1.sol

```diff
@@ -0,0 +1,67 @@
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+
+import "../libraries/ERC20.sol";
+import "../interfaces/IERC20Extended.sol";
+
+contract MockERC20WithFee1 is ERC20, IERC20Extended {
+    using SafeMath for uint256;
+
+    constructor(
+        string memory _name,
+        string memory _symbol,
+        uint256 _totalSupply
+    ) public ERC20(_name, _symbol) {
+        _mint(msg.sender, _totalSupply);
+    }
+
+    function _transfer(
```

Comment:
how come internal function here?
above all external?

if there is a reason. would document it.
or else move according to convention

Reply:
Ok